### ISSUE TITLE
create `gh-pages` branch

### ISSUE THREAD

**Issue by gipsy-king** [Civcraft/RealisticBiomes#23]
copied my spreadsheet from txapu.com to github pages, would look like this http://gipsy-king.github.io/RealisticBiomes/spreadsheet/

I need someone to create a branch called exactly `gh-pages` so that I can make a PR to it:
https://github.com/gipsy-king/RealisticBiomes/tree/gh-pages

**Comment by ttk2** [comment]
this is for documentation stuff?

**Comment by drjawa** [comment]
Hmm, thinking on the issue of documentation fragmentation here but would
prefer everything be in the wiki, but this presentation is superior to the
current google docs spreadsheet.

I guess it couldn't hurt to replace the google docs link in the wiki with a
civcraft.github.io/realisticbiomes link instead.

tldr; documentation of growth rates and all that

**Comment by gipsy-king** [comment]
yea right now it's hosted at txapu.com which is not ideal. with this gh-pages branch thing, it both hosts the spreadsheet AND the source of it, here on github.

**Comment by ttk2** [comment]
that sounds good, roruke make it so

**Comment by WildWeazel** [comment]
https://github.com/Civcraft/RealisticBiomes/tree/gh-pages
